torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -180,11 +180,12 @@ void initNvFuserPythonBindings(PyObject* module) {
       .def(
           "define_constant",
           [](nvfuser::FusionDefinition& self,
-             c10::complex<double> val) -> nvfuser::Scalar* {
+             std::complex<double> val) -> nvfuser::Scalar* {
```

Comment:
Did you mean for this to be `std::complex` instead of `c10::complex`?

Reply:
Yes. With std::complex<double>, python complex doesn't resolve to this overload, but went for the other function with `bool val` instead. And that's the root cause of the test failure earlier.